jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Switch.java

```diff
@@ -61,6 +68,109 @@ public boolean isEnabled() {
         return !hasClass("v-input--is-disabled");
     }
 
+    @JDIAction("Get {name}'s color")
+    public String hasColor() {
+        return css("color");
+    }
+
+    @JDIAction("Get {name}'s background color")
+    public String hasBackgroundColor() {
+        return css("background-color");
+    }
+
+    @JDIAction("Get {name}'s slot background color")
+    public String hasSlotsBackgroundColor() {
+        return core().find(".v-input__slot").css("background-color");
+    }
+
+    @JDIAction("Check that {name} has light theme")
+    public Boolean hasLightTheme() {
```

Comment:
Please use HasTheme interface instead of hasLightTheme and add assert for dark theme